src/desktop/Window.cpp

```diff
@@ -836,6 +836,8 @@ void CWindow::createGroup() {
         g_pCompositor->updateWorkspaceSpecialRenderData(workspaceID());
         g_pLayoutManager->getCurrentLayout()->recalculateMonitor(m_iMonitorID);
         g_pCompositor->updateAllWindowsAnimatedDecorationValues();
+
+        g_pEventManager->postEvent(SHyprIPCEvent{"togglegroup", "1"});
```

Comment:
I would also add a window handle as another argument (`handle,state`) so that one can know which window was toggled

Reply:
In the case of creating group when 1 should be returned I understand your point. But what about when user is destroying the window group containing more than one windows. 
Should `togglegroup` event return 0 and empty string (something like this `"",0`) or 0 with array of window addresses inside the handle argument?
e.g `"0x64cea2525760,0x64cea2522380",0`

Reply:
no need for quotes. Make `state` first then and the rest is a list of windows.

Reply:
before we proceed this pr I realized that I also need ipc event for `moveintogroup` and `moveoutofgroup`  commands. I think they can be done in the following way
`moveintogroup` will give single window address which was merged in a group, e.g `0x64cea2525760`
`moveoutofgroup` will also give single window address which was separated from a group

What do you think about this?  and should I create a new PR for this or merge inside this one?

Reply:
sounds ok

Reply:
Implementation and squashing commits done, I think its ready to merge if you approve